service/controller/v19/adapter/host_post_route_tables.go

```diff
@@ -27,20 +27,29 @@ func (i *HostPostRouteTablesAdapter) Adapt(cfg Config) error {
 		return microerror.Mask(err)
 	}
 
+	var tenantPrivateSubnetCidrs []string
+	{
+		for _, az := range key.StatusAvailabilityZones(cfg.CustomObject) {
+			tenantPrivateSubnetCidrs = append(tenantPrivateSubnetCidrs, az.Subnet.Private.CIDR)
+		}
+	}
+
 	// private routes.
 	for _, routeTableName := range cfg.CustomObject.Spec.AWS.VPC.RouteTableNames {
 		routeTableID, err := routeTableID(routeTableName, cfg)
 		if err != nil {
 			return microerror.Mask(err)
 		}
-		rt := HostPostRouteTablesAdapterRouteTable{
-			Name:         routeTableName,
-			RouteTableID: routeTableID,
-			// Requester CIDR block, we create the peering connection from the guest's private subnet.
-			CidrBlock:        key.PrivateSubnetCIDR(cfg.CustomObject),
-			PeerConnectionID: peerConnectionID,
+		for _, cidrBlock := range tenantPrivateSubnetCidrs {
+			rt := HostPostRouteTablesAdapterRouteTable{
+				Name:         routeTableName,
+				RouteTableID: routeTableID,
```

Comment:
Is it fine when for all AZs all route tables have the same name and ID?

Reply:
The name here could be better. I was wondering the same thing when reading code here, but what happens actually is that these are just used to create routes to these referenced routing tables: https://github.com/giantswarm/aws-operator/blob/master/service/controller/v19/templates/cloudformation/hostpost/route_tables.go

...unless I missed something. Naturally this must be verified, but if this is the case, I think this would be clearer when renaming `HostPostRouteTablesAdapterRouteTable` -> `HostPostRouteTablesAdapterRoute`.

Reply:
+1 on naming it `HostPostRouteTablesAdapterRoute`. This is also the resource type.

Reply:
Renamed

Reply:
Done